netctl/netctl.go

```diff
@@ -237,23 +235,18 @@ func listNetworks(ctx *cli.Context) {
 	} else {
 		writer := tabwriter.NewWriter(os.Stdout, 0, 2, 2, ' ', 0)
 		defer writer.Flush()
-		writer.Write([]byte("Tenant\tNetwork\tPublic\tEncap\tSubnet\tGateway\n"))
-		writer.Write([]byte("------\t-------\t------\t-----\t------\t-------\n"))
+		writer.Write([]byte("Tenant\tNetwork\tEncap\tTag\tSubnet\tGateway\n"))
```

Comment:
I made it look like this below.
Note that description match. but, arg option is in abbreviated form

```
vagrant@netplugin-node1:~$ netctl net create --help
NAME:
   netctl net create - Create a network

USAGE:
   netctl net create [command options] [network]

OPTIONS:
   --tenant, -t "default"	Name of the tenant
   --encap, -e "vxlan"		Encap type (vlan or vxlan)
   --pkt-tag, -p 		Packet tag (Vlan/Vxlan ids)- REQUIRED
   --subnet, -s 		Subnet CIDR - REQUIRED
   --gateway, -g 		Gateway - REQUIRED
   
vagrant@netplugin-node1:~$ netctl net list
Tenant   Network     Encap type  Packet tag  Subnet          Gateway
------   -------     ----------  ----------  -------         ------
default  private     vxlan       1001        10.1.0.0/16     10.1.254.254
default  public      vlan        1           192.168.1.0/24  192.168.1.254
default  contiv-net  vxlan       1005        20.1.1.0/24     20.1.1.1
```